ccp/lib/_print.py

```diff
@@ -0,0 +1,15 @@
+"""
+This library util module has _print method defined
+which can be used to print messages on stdout realtime
+and not being blocked on stdout buffer for showing the messages.
+"""
+
+import sys
+
+
+def _print(msg):
```

Comment:
Maybe add verbose parameter here which is default True, so it can be used with those cases as well

Reply:
I am planning to add logging mechanism here.

If set to a file will export to file or to console, will add all those options then.